lib/modules/dashboard.js

```diff
@@ -241,7 +241,7 @@ function attachEditComponent() {
 		<div class="editWidget">Edit widget</div>
 		<div id="editRedditFormContainer" class="editRedditForm">
 			<form id="editRedditForm">
-				<input type="text" id="editReddit">
+				<input type="textarea" id="editReddit">
```

Comment:
This should be literally `<textarea id="editReddit"></textarea>`, since [`textarea` is not a valid type of `input`](https://codepen.io/andytuba/pen/bxpVLd?editors=1100).